pkg/cli/admin/prune/images/images.go

```diff
@@ -355,7 +357,7 @@ func (o PruneImagesOptions) Run() error {
 	}
 
 	allDCs, err := o.AppsClient.DeploymentConfigs(o.Namespace).List(context.TODO(), metav1.ListOptions{})
-	if err != nil {
+	if err != nil && !kerrors.IsNotFound(err) {
```

Comment:
As far as I understand, `allDCs` will not be nil just like https://github.com/openshift/oc/blob/61cc186c35bd568e16dd62712956e27c40cd65a2/pkg/cli/admin/prune/images/images.go#L335 to pass to https://github.com/openshift/oc/blob/68e3d16b4426d95374701fbb0b23f0448b04f29c/pkg/cli/admin/prune/imageprune/prune.go#L658.

Reply:
Thanks for the quick review @ardaguclu.
Do you mean that it's safe to pass `allDCs` around, thus we do not need to further modify the code?

Reply:
According to the `Build` resource, it seems safe but I think, we need to ensure that it won't panic

Reply:
Indeed. I haven't been able to try this out o a running cluster yet (getting errors on bootstrap), will ping IR QE to help us.